<launch>
  <node pkg = "ransac_project" type = "nodeControl" name = "nodeControl" output="screen" clear_params="true">
    <param name = "KPT" value = "20.0" />
    <param name = "KIT" value = "0.1" />
    <param name = "KRT" value = "1.0" />
    <param name = "KVT" value = "0.1" />
    <param name = "length" value = "4.0" />
    <param name = "rampTime" value = "5.0" />
    <!--param name = "platform" value = 'pioneer' /-->
    <param name = "platform" value = 'vero' />
    <param name = "maxLinearVel" value = "0.3" />
  </node>
</launch>
